src/Event/NewConferenceEvent.php

```diff
@@ -87,7 +85,7 @@ public function buildAttachment(): array
         if ($this->conference->getParticipations()->count() > 0) {
             $starfleetLinkField = SlackNotifier::SHORT_FIELD;
             $starfleetLinkField['title'] = 'Starfleet link  🚀';
-            $starfleetLinkField['value'] = $this->router->generate('conferences_show', [
+            $starfleetLinkField['value'] = $router->generate('conferences_show', [
```

Comment:
Ohhhh, I see why the router was in this class before.
Don't modify this code for now, but It is absolutely not the place for this method :) 
It's not the responsibility to the event to know how to transform it to a slack attachment. It's the responsibility of the slack notifier.

TL;DR: don't do it now, but theses method should be move to the slack notifier (you can do it later, or not, up to you :) )